### DISCUSSION TITLE
How do I use diamond getseq?

### DISCUSSION THREAD

**Discussion by astridfilt** [bbuchfink/diamond#563] [Q&A]
Hi :) 

I have aligned a bunch of peptides against a diamond database of uniprot sequences using diamond blastp. Now, I want to retrieve the sequences from the diamond database, and I thought I could use diamond getseq for that. But so far, it is not working for me. The commands I have tried are:
diamond getseq --db my_diamond_db.dmnd --seq ["seqID_1" "seqID_2" "seqID_3"]
and 
diamond getseq --db my_diamond_db.dmnd --seq "seqID_1" "seqID_2" "seqID_3"
and
diamond getseq --db my_diamond_db.dmnd --seq "seqID_1" "seqID_2" "seqID_3" --out getseq_out

The seqID's are the IDs that are reported as sseqid when I run diamond blastp. The last command did create an output file, but it was empty.

Thank you in advance :)

**Comment by astridfilt** [comment]
I found an alternative to getseq, which is seqtk subseq. That gave me what I needed. But I am still curious if I could do it with getseq :)

**Comment by bbuchfink** [comment]
The arguments to `--seq` are numbers, e.g. you can use `--seq 1 3` to get the first and the third sequence in the database. You can't do it based on accessions but if you use a BLAST database, another good way for doing this is `blastdbcmd`.
